labels.go

```diff
@@ -186,6 +190,46 @@ func PrevLabel(s string, n int) (i int, start bool) {
 	return 0, n > 1
 }
 
+// compares domains according to the canonical ordering specified in RFC4034
+// returns an integer value similar to strcmp
+// names have to have equal casing!
```

Comment:
I would like this to be not true, it should be case agnostic

Reply:
Whoops, it *is* case agnostic now after starting using `equal`, some of the test cases would fail otherwise. That comment will have to be updated.